key/keyutil/keyutil.go

```diff
@@ -0,0 +1,51 @@
+package keyutil
+
+import (
+	"encoding/binary"
+	"math/rand"
+	"strconv"
+
+	"github.com/plprobelab/go-kademlia/key"
+)
+
+var rng = rand.New(rand.NewSource(299792458))
+
+// Random returns a KadKey of length l populated with random data.
```

Comment:
here, `length` refers to the number of bytes instead of the number of bits. I haven't checked there rest but the usage of `length` in the context of a KadKey should be consistent throughout the code.

Alternatively we could use:

- `size` -> bytes
- `length` -> bits

Reply:
good catch